test/common/testlib.py

```diff
@@ -1604,9 +1604,12 @@ def allow_browser_errors(self, *patterns):
             self.allowed_console_errors.append(p)
 
     def check_browser_errors(self):
-        if not self.browser:
+        b = self.browser
+
+        if not b:
```

Comment:
I start to prefer assignment expressions for cases like these, but that's a matter of taste.

Reply:
These are fine if the scope of `b` is limited to the "then" part. But here it isn't -- it's our standard "local short alias for the whole function" pattern.

Reply:
This is gone now.